### DISCUSSION TITLE
Integrating kafka with channels

### DISCUSSION THREAD

**Discussion by aryaniyaps** [django/channels#1641] [Show and tell] [answered]
I've done a detailed write-up on kafka-integration with channels
https://gist.github.com/aryan340/da071d027050cfe0a03df3b500f2f44b

this addresses #1637 and hopefully solves it.

**Comment by carltongibson** [comment]
Hi @CosmicReindeer — that's a lovely write-up: really nice!

I'd happily accept a link to it from the community page of the docs if you'd like to make that PR.

**Comment by aryaniyaps** [comment, marked as answer]
Done! (PR is [here](https://github.com/django/channels/pull/1644))

**Comment by hrichards** [comment]
The Gist link is now dead.  Is there an updated reference to this write-up anywhere?
